ROOTFS/etc/waggle/sanity/warning/check_hanwha_camera.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
+
+import sys
+import json
+import urllib.error, urllib.request
+import time
```

Comment:
you dont seem to be using this `import`. remove?

Reply:
`import time` is removed!